mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -1365,6 +1365,24 @@ func resourceContainerCluster() *schema.Resource {
 					},
 				},
 			},
+			"mesh_certificates": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Computed:    true,
+				ForceNew:    true,
```

Comment:
It looks like this & enable_certificates should be updatable: https://cloud.google.com/traffic-director/docs/security-proxyless-setup#create-cluster - just want to double-check that they should be ForceNew?